cmd/neofs-cli/modules/acl/extended/create.go

```diff
@@ -67,35 +68,35 @@ func createEACL(cmd *cobra.Command, _ []string) {
 
 	containerID := cid.New()
 	if err := containerID.Parse(cidArg); err != nil {
-		cmd.Printf("invalid container ID: %v", err)
-		return
+		cmd.Printf("invalid container ID: %v\n", err)
+		os.Exit(1)
 	}
 
 	rulesFile, err := getRulesFromFile(fileArg)
 	if err != nil {
-		cmd.Printf("can't read rules from file : %v", err)
-		return
+		cmd.Printf("can't read rules from file: %v\n", err)
```

Comment:
`cmd.PrintErrf` here and in other places?

Reply:
Done.